todo_app/tests/test_integration.py

```diff
@@ -28,60 +28,15 @@ def test_index_page(mock_get_requests, client):
     mock_get_requests.side_effect = mock_get_lists
```

Comment:
You're still mocking the lists, even though nothing makes a request any more! If you follow the mocking suggestion above, it would be nice to switch this to instead add some items to the fake database